core/templates/dev/head/components/share/sharing_links_directive.html

```diff
@@ -25,6 +25,26 @@
       </a>
     </li>
   </ul>
+
+  <ul class="oppia-sharing-links" layout="<[layoutType]>" layout-align="<[layoutAlignType]>" ng-if="shareType==='collection'">
```

Comment:
There could be less repetition. Why not consolidate, and just put the ng-if around the embed button? You can define additional params as needed for the hrefs, e.g. url=<[serverName]>/<[activityType]>/<[activityId]>.

The point here is to reduce unnecessary duplication.